components/create_post/create_post.jsx

```diff
@@ -464,16 +463,7 @@ export default class CreatePost extends React.Component {
         }
     }
 
-    postMsgKeyPress = (e) => {
-        const ctrlOrMetaKeyPressed = e.ctrlKey || e.metaKey;
-        if (!UserAgent.isMobile() && ((this.props.ctrlSend && ctrlOrMetaKeyPressed) || !this.props.ctrlSend)) {
-            if (Utils.isKeyPressed(e, KeyCodes.ENTER) && !e.shiftKey && !e.altKey) {
-                e.preventDefault();
-                ReactDOM.findDOMNode(this.refs.textbox).blur();
```

Comment:
It has been there from the first github commit available in mattermost-server (the old platform repo, where is the code since 14th Jun, 2015). My concern is, was it there for any reason?, can we delete that?

Reply:
Back then, we only used the `onPress` event for this. At some point, we switched to using `onPress` for some things and `onKeyDown` for others, and [when ctrl+enter to send was added](https://github.com/mattermost/mattermost-server/pull/1196/files#diff-172243f3b7d83f0a243e8539665916aeR213), we made one method call the other for some reason. I'm pretty sure that caused `this.handleSubmit` to be called twice when you hit ctrl+enter (once for the keyDown and once for the keyPress), but the code that prevents you from double-tapping enter and sending twice would prevent that from happening

This cleans that up by only using the one event instead of both to detect keyboard shortcuts.